requirements.txt

```diff
@@ -87,13 +91,18 @@ tzlocal==1.2
 ujson==1.35
 urllib3==1.22
 utaw==0.2.0
+vault-tools==0.6.12
 venusian==1.1.0
 webcolors==1.7
 WebOb==1.7.4
 websocket-client==0.44.0
 whichcraft==0.4.0
 wsgicors==0.7.0
 yarl==1.1.1
+yelp-cgeom==1.3.1
 yelp-clog==2.12.1
+yelp-logging==1.0.37
+yelp_meteorite
```

Comment:
This isn't opensource right, so won't this break on travis? We have another requirements file for the yelpy stuff

Reply:
Yep, looks like this is handled by https://github.com/Yelp/paasta/blob/master/yelp_package/extra_requirements_yelp.txt, which already has these requirements.